opr/primer.py

```diff
@@ -217,6 +218,36 @@ def gc_clamp(self, _):
     def gc_clamp(self, _):
         raise OPRBaseError(PRIMER_NOT_REMOVABLE_ATTRIBUTE_ERROR)
 
+    @property
+    def single_runs(self):
+        """
+        Calculate Single Runs of the primer.
+
+            Run length refers to how many times a single base is repeated consecutively in a primer
```

Comment:
1. Run length refers to how many times a single base is repeated consecutively in <s>a</s> the primer
2. Fix white-space bug